### PR TITLE
WIP: Rubocop

### PR DESCRIPTION
**Pull request by luizkowalski**

### CODE REVIEW

File: lib/cloudwatchlogger/client.rb [hunk 1]
```diff
@@ -10,7 +9,7 @@ def self.new(credentials, log_group_name, log_stream_name = nil, opts = {})
         raise LogGroupNameRequired, 'log_group_name is required'
       end
 
-      CloudWatchLogger::Client::AWS_SDK.new(credentials, log_group_name, log_stream_name, opts)
+      CloudWatchLogger::Client::AwsSDK.new(credentials, log_group_name, log_stream_name, opts)
```
Comment (luizkowalski) [inline lib/cloudwatchlogger/client.rb:12]: not sure about this class name. for its ruby gem, AWS uses `Aws::` as the namespace, but I'm not 100% comfortable with `SDK` here

File: lib/cloudwatchlogger/client/aws_sdk/threaded.rb [hunk 2]
```diff
@@ -48,7 +47,7 @@ def initialize(credentials, log_group_name, log_stream_name, opts = {})
 
           super do
             loop do
-              connect!(opts) if @client.nil?
+              setup_resources! if @client.nil?
```
Comment (luizkowalski) [inline lib/cloudwatchlogger/client/aws_sdk/threaded.rb:52]: changed the method name to reflect what the method actually does

File: lib/cloudwatchlogger/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module CloudWatchLogger
-  VERSION = '0.2.1'.feeze
+  VERSION = '0.2.1'.freeze
```
Comment (luizkowalski) [inline lib/cloudwatchlogger/version.rb:2]: how did I miss this

**Comment by zshannon** [comment]
Please send two PRs, one that's exclusively Rubocop stuff, and if you want one with these other changes, though I'll advise now that I'm presently skeptical of their necessity or benefits.

**Comment by luizkowalski** [comment]
what “other changes” you mean? the mehod rename for example?

**Comment by zshannon** [comment]
Yup! I'd like to see a clean rubocop PR that's its config file, adding it as a dev dep gem, README about running it, and whatever the results are of its first run.

**Pull request closed without merging**